### ISSUE TITLE
VEP plugin for loss of function transcript effect estimator

### ISSUE THREAD

**Issue by pontikos** [vplagnol/pipelines#28]
As suggested by @APLevine, we should add this Plugin https://github.com/konradjk/loftee

**Comment by pontikos** [comment]
I've added it to:
```
/cluster/project8/vyp/AdamLevine/software/ensembl/Plugins
```
Let's see see if works.

**Comment by pontikos** [comment]
It runs ok but I'm seeing this:

2015-01-31 15:59:12 - Calculating consequences
[fai_load] build FASTA index.
[fai_build] fail to open the FASTA file human_ancestor.fa.rz
[fai_load] fail to open FASTA index.
Could not load fai index of human_ancestor.fa.rz
[fai_load] build FASTA index.
[fai_build] fail to open the FASTA file human_ancestor.fa.rz
[fai_load] fail to open FASTA index.
Could not load fai index of human_ancestor.fa.rz

which I think is linked to loftee:
https://github.com/chapmanb/bcbio-nextgen/issues/468
https://github.com/konradjk/loftee/blob/master/LoF.pm#L295

**Comment by vplagnol** [comment]
Yes it apparently needs the ancestral alleles. Which makes sense. Are these
files available for build 38?

**Comment by pontikos** [comment]
Not sure.
All I can find is this:
ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase1/analysis_results/supporting/ancestral_alignments/

**Comment by vplagnol** [comment]
Another hg38 struggle. Perhaps not a great decision. Anyway ignore for now
I think and let's ask the authors if they have updated files.

**Comment by pontikos** [comment]
Ok we will leave it out for now.
